querydsl-core/src/test/java/com/querydsl/core/serialization/SerializerBaseTest.java

```diff
@@ -43,3 +42,26 @@ public void test() {
     }
 
 }
+
+class DummySerializer extends SerializerBase<DummySerializer> {
```

Comment:
I'd prefer to have this as a top level class with package private visibility. I try to follow the one top level class per file rule.